src/lumigo_tracer/lumigo_utils.py

```diff
@@ -386,7 +386,7 @@ def report_json(
         internal_analytics_message("report: socket.timeout")
     except Exception as e:
         if should_retry:
-            get_logger().info(f"Could not report to {host}. Retrying.", exc_info=e)
+            get_logger().info(f"Could not report to {host}: ({str(e)}). Retrying.")
```

Comment:
why is this change? isn't better to let the info function handle the exception display?

Reply:
This is a legitimate flow, and the exception is really terrifying. 
Generally you're right that we better have more data, but here I prefer to not bomb the logs :)